CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+Contributing
+============
+
+We'd love your help making M3DB great!
```

Comment:
This is for M3DB, change or delete this file?

Reply:
This file is in a lot of repos under m3db/, like m3x m3cluster m3db, since all the repos are still under the org of m3db, I guess just leave it here for now?

Reply:
Hm on line 19 it talks about submitting issues to the m3db/m3db repo. I'd suggesting changing M3DB to M3Msg where needed. or just deleting this file.

Reply:
sure, can delete the file